src/components/breadcrumbs/index.tsx

```diff
@@ -1,7 +1,7 @@
 import Link from "next/link";
 
 type BreadcrumbsInfo = {
-  name: string;
+  name?: string;
```

Comment:
is this required as part of this change or something else you noticed?

Reply:
That was a complain that name could be undefined